spec/policies/core_induction_programme_policy_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe CoreInductionProgrammePolicy, type: :policy do
+  subject { described_class.new(user, core_induction_programme) }
+  let(:core_induction_programme) { create(:core_induction_programme) }
+
+  context "accessing cip views as admin" do
+    let(:user) { create(:user, :admin) }
+    it { is_expected.to permit_action(:show) }
+    it { is_expected.to permit_action(:index) }
+  end
+
+  context "accessing cip views as an early career teacher" do
+    let(:user) { create(:user, :early_career_teacher, { core_induction_programme: core_induction_programme }) }
+    it { is_expected.to permit_action(:show) }
+    it { is_expected.not_to permit_action(:index) }
```

Comment:
I think you can write 
```
it { is_expected.not_to permit_action(:index) }
```
as
```
it { is_expected.to forbid_action(:index) }
```
I think it's slightly easier on the brain, but up to you if you want to write it this way :)